### ISSUE TITLE
wrong import in types.d.ts

### ISSUE THREAD

**Issue by 435352980** [sanusart/react-dropdown-select#68]
https://github.com/sanusart/react-dropdown-select/blob/9b93c6ef5059da05bab98b9798c8e03ae9d7ef88/types.d.ts#L1

should change to
```js
import React,{ SyntheticEvent, KeyboardEvent } from 'react'; 
```

**Comment by 435352980** [comment]
And `portal` prop not receive string eg. `portal={document.body}` instead of `portal="body"`

**Comment by sanusart** [comment]
Thank you, published version 3.11.1
